src/eks-mcp-server/README.md

```diff
@@ -0,0 +1,464 @@
+    style D fill:#fbb,stroke:#333,stroke-width:2px
+    style E fill:#fbf,stroke:#333,stroke-width:2px
+    style F fill:#dff,stroke:#333,stroke-width:2px
+```
+
+The server uses **CloudFormation** templates for EKS cluster creation, ensuring consistent, repeatable cluster deployments with proper networking, security, and node configurations. It also implements a modular architecture with clear separation of concerns between EKS operations and Kubernetes resource management.
+
+## Key Capabilities
+
+- **EKS Cluster Management**
+  - Create EKS clusters with dedicated VPCs and proper networking
+  - Configure Kubernetes version, node pools, and other cluster settings
+  - Use CloudFormation for reliable and repeatable cluster creation
+
+- **Kubernetes Resource Management**
+  - Create, replace, patch, delete, and read Kubernetes resources
+  - Apply YAML manifests to EKS clusters
+  - List resources with filtering by namespace, labels, and fields
+
+- **Application Deployment**
+  - Generate Kubernetes manifests for application deployment
+  - Configure deployments with customizable parameters
+  - Create services for external access
+
+- **Operational Support**
+  - Retrieve logs from pods with filtering options
+  - Get events related to specific Kubernetes resources
+  - Monitor cluster and resource status
+
+- **CloudWatch Integration**
+  - Retrieve CloudWatch logs for Kubernetes resources
+  - Get CloudWatch metrics for cluster monitoring
+  - Support for different log types and resource types
+
+## Prerequisites
+
+1. Install `uv` from [Astral](https://docs.astral.sh/uv/getting-started/installation/) or the [GitHub README](https://github.com/astral-sh/uv#installation)
+2. Install Python using `uv python install 3.10`
+3. AWS account with permissions to create and manage EKS resources
+
+## Setup
+
+### Installation
+
+Configure the MCP server in your MCP client configuration (e.g., for Amazon Q Developer CLI, edit `~/.aws/amazonq/mcp.json`):
+
+```json
+{
+  "mcpServers": {
+    "awslabs.eks-mcp-server": {
+      "command": "uv",
+      "args": [
+        "--directory",
+        "/path/to/awslabs/eks_mcp_server",
+        "run",
+        "server.py"
+      ],
+      "env": {
+        "AWS_PROFILE": "your-aws-profile",
+        "AWS_REGION": "us-east-1",
+        "FASTMCP_LOG_LEVEL": "INFO"
+      }
+    }
+  }
+}
+```
+
+If you would like to specify additional options, you can pass them to the args:
+
+```json
+{
+  "mcpServers": {
+    "awslabs.eks-mcp-server": {
+      "command": "uv",
+      "args": [
+        "--directory",
+        "/path/to/awslabs/eks_mcp_server",
+        "run",
+        "server.py",
+        "--allow-write", // Optional parameter if you would like to allow MCP to take mutating actions (i.e. Create/Update/Delete Resource)
+        "--allow-sensitive-data-access" // Optional parameter if you would like to allow MCP to access sensitive data such as logs, events, and Kubernetes Secrets
```

Comment:
```suggestion
        "--allow-write",
        "--allow-sensitive-data-access"
```
Remove comments from json code snippet, as user would copy and paste the code and this will cause errors in tools reading the json config, as the comments make the json section invalid